velox/functions/prestosql/tests/JsonFunctionsTest.cpp

```diff
@@ -100,6 +101,77 @@ TEST_F(JsonFunctionsTest, jsonFormat) {
   facebook::velox::test::assertEqualVectors(expected, result);
 }
 
+TEST_F(JsonFunctionsTest, jsonParse) {
+  const auto jsonParse = [&](std::optional<std::string> value) {
+    return evaluateOnce<StringView>("json_parse(c0)", value);
+  };
+
+  const auto jsonParseWithTry = [&](std::optional<std::string> value) {
+    return evaluateOnce<StringView>("try(json_parse(c0))", value);
+  };
+
+  EXPECT_EQ(jsonParse(std::nullopt), std::nullopt);
+  EXPECT_EQ(jsonParse(R"(true)"), "true");
+  EXPECT_EQ(jsonParse(R"(null)"), "null");
+  EXPECT_EQ(jsonParse(R"(42)"), "42");
+  EXPECT_EQ(jsonParse(R"("abc")"), R"("abc")");
+  EXPECT_EQ(jsonParse(R"([1, 2, 3])"), "[1, 2, 3]");
+  EXPECT_EQ(jsonParse(R"({"k1":"v1"})"), R"({"k1":"v1"})");
+  EXPECT_EQ(jsonParse(R"(["k1", "v1"])"), R"(["k1", "v1"])");
+
+  VELOX_ASSERT_THROW(jsonParse(R"({"k1":})"), "expected json value");
+  VELOX_ASSERT_THROW(jsonParse(R"({:"k1"})"), "expected json value");
+  VELOX_ASSERT_THROW(jsonParse(R"(not_json)"), "expected json value");
+
+  EXPECT_EQ(jsonParseWithTry(R"(not_json)"), std::nullopt);
+  EXPECT_EQ(jsonParseWithTry(R"({"k1":})"), std::nullopt);
+  EXPECT_EQ(jsonParseWithTry(R"({:"k1"})"), std::nullopt);
+
+  auto elementVector = makeNullableFlatVector<StringView>(
+      {R"("abc")", R"(42)", R"({"k1":"v1"})", R"({"k1":})", R"({:"k1"})"});
+  auto resultVector =
+      evaluate("try(json_parse(c0))", makeRowVector({elementVector}));
+
+  auto expectedVector = makeNullableFlatVector<StringView>(
+      {R"("abc")", "42", R"({"k1":"v1"})", std::nullopt, std::nullopt});
+  facebook::velox::test::assertEqualVectors(expectedVector, resultVector);
+
+  auto data = makeRowVector({makeConstant(R"("k1":)", 2)});
+  expectedVector =
+      makeNullableFlatVector<StringView>({std::nullopt, std::nullopt});
+  facebook::velox::test::assertEqualVectors(
```

Comment:
I removed "facebook::" prefix and got this file to compile without error. I suggest you remove this prefix throughout this file.